src/macros.jl

```diff
@@ -436,14 +436,37 @@ Add additional columns or keys based on keyword arguments.
 
 * `::AbstractDataFrame` or `::GroupedDataFrame`
 
+Inputs to `@transform` can come in two formats. One
+can create new columns in a single block, where each
```

Comment:
```suggestion
Inputs to `@transform` can come in two formats.
New columns can be created either in a single block, where each
```
(I first read "one" as meaning "one way is to".)